esmvaltool/recipes/hydrology/recipe_marrmot.yml

```diff
@@ -0,0 +1,72 @@
+# ESMValTool
+# recipe_marrmot.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the MARRMoT hydrological model.
+
+    MARRMoT is the Modular Assessment of Rainfall-Runnoff Models Toolbox ...
+    https://www.geosci-model-dev.net/12/2463/2019/
+  authors: ['kalverla_peter', 'camphuijsen_jaro', 'alidoost_sarah']
+  projects: ['ewatercycle']
+  references: ['acknow_project']
+
+preprocessors:
+  extract_catchment: &catchment
+    extract_shape:
+      # Lumped model: needs catchment-aggregated input data
+      shapefile: meuse_hydrosheds.shp  # In aux (config-user.yml)
+      method: contains
+      crop: true
+  daily_mean: &mean
+    daily_statistics:
+      operator: mean
+  daily:
+    <<: *catchment
+  hourly:
+    <<: *catchment
+    <<: *mean
+
+diagnostics:
+  diagnostic_daily:
+    description: marrmot input preprocessor for daily data
+    additional_datasets:
+      - {dataset: ERA-Interim, project: OBS6, tier: 3, type: reanaly, version: 1}
+    variables:
+      tas: &var_daily
+        mip: day
+        start_year: 1990
+        end_year: 2018
+        preprocessor: daily
+      pr: *var_daily
+      # MARRMoT requires PET. Thus, we might also need psl, rsds, rsdt
+      psl: *var_daily
+      rsds: *var_daily
+      rsdt:
+        <<: *var_daily
+        mip: CFday
+    scripts:
+      script:
+        script: hydrology/marrmot.py
+
+  diagnostic_hourly:
+    description: marrmot input preprocessor for hourly data
+    additional_datasets:
+      - {dataset: ERA5, project: OBS6, tier: 3, type: reanaly, version: 1}
+    variables:
+      tas: &var_hourly
+        mip: E1hr
+        start_year: 1990
+        end_year: 2018
+        preprocessor: hourly
+      pr: *var_hourly
+      # MARRMoT requires PET. Thus, we might also need psl, rsds, rsdt
+      psl: *var_hourly
+      rsds: *var_hourly
+      rsdt: *var_hourly
+      # Alternatively, we might use (but this is inconsistent with ERA-Interim):
+      # evspsblpot: *var_hourly
```

Comment:
It looks like this is not implemented in the code

Reply:
Thanks, it is removed.